backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -405,17 +407,17 @@ public String getSpecimenSourceSiteCode() {
 
 	@JsonProperty("Specimen_type_code")
 	public String getSpecimenTypeCode() {
-		return anteriorNaresSwabCode;
+		return device.getSwabType();
 	}
 
 	@JsonProperty("Instrument_ID")
 	public String getInstrumentID() {
-		return testEvent.getDeviceType().getInternalId().toString();
+		return device.getInternalId().toString();
```

Comment:
I hadn't realized this was our internal ID—why does anybody care about that? Is this supposed to allow for individual instruments to be sent? It seems like it should be the LOINC code, but I assume that's somewhere else, so I don't know what this field is for. 😕